{
    "docs": "https://docs.ansible.com/ansible-core/2.18/dev_guide/testing/sanity/no-assert.html", 
    "results": [
        {
            "message": "The test `ansible-test sanity --test no-assert` [[explain](https://docs.ansible.com/ansible-core/2.18/dev_guide/testing/sanity/no-assert.html)] failed with 1 error:", 
            "output": "plugins/module_utils/dsldap.py:119:9: raise AssertionError instead of: assert"
        }
    ], 
    "verified": false
}
